js/main.js

```diff
@@ -1,7 +1,7 @@
 const anchors = document.querySelectorAll('a[href*="#"]');
 
-for (let anchor of anchors) {
-  anchor.addEventListener("click", function (e) {
```

Comment:
better to use `for of` for iterations over objects. `document.querySelectorAll` returns NodeList, so you can iterate over it using `.forEach`